### ISSUE TITLE
Evaluating variance in output

### ISSUE THREAD

**Issue by pengnam** [kyle-dorman/bayesian-neural-network-blogpost#7]
![image](https://user-images.githubusercontent.com/28092058/55728002-d4be5f00-5a45-11e9-8426-40cd5d5e3d36.png)

The model trains with both logits_variance and softmax as the output. 

softmax is trained with the classification value

What is logits variance trained with?

**Comment by pranavpandey2511** [comment]
@pengnam  Its kind of a unsupervised learning. We use Monte carlo simulation to produce estimated true values for the variance of any given input and the prediction of NN is trained with those values.
